acceptance-test/src/acceptance-test/java/org/zalando/nakadi/repository/db/SubscriptionTokenListerTest.java

```diff
@@ -175,7 +180,7 @@ public void whenListSubscriptionsByMultipleEventTypesThenOk() throws ServiceTemp
                 .collect(toList());
 
         final List<Subscription> subscriptions = subscriptionTokenLister.listSubscriptions(ImmutableSet.of(et1, et2),
-                Optional.empty(), null, 10).getItems();
+                Optional.empty(), Optional.empty(), null, 10).getItems();
```

Comment:
Same here: only empty values used for the new parameter.

Reply:
Would an acceptance test justify few unit tests?

Reply:
I would happily exchange an acceptance test for any number of mocked tests.  A unit test doesn't hurt, but if it requires mocking, let's go for acceptance test instead. ;)